### PR TITLE
planarity: init at 3.0.0.5

### PR DESCRIPTION
**Pull request by timokau**
###### Motivation for this change

Package planarity.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by davidak** [comment]
Thanks for your contribution. It builds and runs also on macOS.

File: pkgs/development/libraries/science/math/planarity/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      sha256 = "1nqjc4clr326imz4jxqxcxv2hgh1sjgzll27k5cwkdin8lnmmil8";
+    })
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/graph-algorithms/edge-addition-planarity-suite;
+    description = "A library for implementing graph algorithms";
+    license = with licenses; [ gpl2 ];
+    maintainers = with maintainers; [ timokau ];
+    platforms = platforms.all;
```
Comment (dotlambda) [inline pkgs/development/libraries/science/math/planarity/default.nix]: `platforms.unix`

File: pkgs/development/libraries/science/math/planarity/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    (fetchpatch {
+      url = "https://github.com/graph-algorithms/edge-addition-planarity-suite/pull/3.patch";
+      sha256 = "1nqjc4clr326imz4jxqxcxv2hgh1sjgzll27k5cwkdin8lnmmil8";
+    })
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/graph-algorithms/edge-addition-planarity-suite;
+    description = "A library for implementing graph algorithms";
+    license = with licenses; [ gpl2 ];
```
Comment (dotlambda) [inline pkgs/development/libraries/science/math/planarity/default.nix]: `licenses.bsd3`
https://github.com/graph-algorithms/edge-addition-planarity-suite/blob/master/LICENSE.TXT

**Comment by dotlambda** [comment]
@GrahamcOfBorg build planarity

**Pull request merged**